registration/models.py

```diff
@@ -359,10 +360,18 @@ class Attendee(models.Model):
     parentEmail = models.CharField(max_length=200, blank=True)
     aslRequest = models.BooleanField(default=False)
 
+    def getFirst(self):
+        if not self.preferredName:
+            return self.firstName
+        else:
+            return self.preferredName
```

Comment:
We are missing test coverage for this path, in `test_model.py`, we could add a test:

```python
class TestAttendee(TestCase):
    def setUp(self):
        self.attendee = Attendee(*TEST_ATTENDEE_ARGS)
    def test_preferredName(self):
        preferredName = "Someone else"
        self.attendee.preferredName = preferredName
        self.assertEqual(self.attendee.getFirst(), preferredName)
```